hokusai/staging.yml

```diff
@@ -108,3 +108,43 @@ spec:
 {%- for cidr_block in cloudflareIpSourceRanges %}
     - {{ cidr_block }}
 {%- endfor %}
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: artsy-wwwify
+    component: web
+    layer: application
+  name: artsy-wwwify-web-internal
+  namespace: default
+spec:
+  ports:
+    - port: 3000
+      protocol: TCP
+      name: http
+      targetPort: 3000
+  selector:
+    app: artsy-wwwify
+    layer: application
+    component: web
+  type: ClusterIP
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: artsy-wwwify
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ cloudflareIpSourceRanges|join(',') }}"
+spec:
+  rules:
+    - host: artsy-wwwify.artsy.net
```

Comment:
@joeyAghion didn't know what DNS name to do for staging?  Perhaps this should also be `artsy.net`?